core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js

```diff
@@ -4,6 +4,62 @@ var isAutoUpdateTurnedON = true;
 var isMemberCellExpanded = {};
 var isMemberRowExpanded = {};
 
+function setClusterStartDate() {
+  var months = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN' , 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'];
+
+  var clusterStartTime = $("#hiddenData").data("clusterstarttime");
+  var dt = new Date(clusterStartTime);
+  var dd = dt.getDate();
+  if ( dd < 10 ) { dd = '0' + dd; }
+
+  var displayDateStr = months[dt.getMonth()] + ' ' + dd + ', ' + dt.getFullYear()
+                     + ' ' + dt.getHours() + ':' + dt.getMinutes() + ':' + dt.getSeconds();
+
+  $("#clusterStartDate").html(displayDateStr);
+  updateClusterUptime();
+}
+
+function updateClusterUptime() {
+  var clusterStartTime = $("#hiddenData").data("clusterstarttime");
+  var start_date = new Date(clusterStartTime);
+  var now_date = new Date();
+
+  var seconds = Math.floor((now_date - start_date)/1000);
+  var minutes = Math.floor(seconds/60);
+  var hours = Math.floor(minutes/60);
+  var days = Math.floor(hours/24);
+
+  hours = hours-(days*24);
```

Comment:
Formatting needs to be corrected. Space before/after - (minus) and *. Also at other places in this file

Reply:
Done